exporter.go

```diff
@@ -305,6 +306,7 @@ func NewRedisExporter(redisURI string, opts ExporterOptions) (*Exporter, error)
 		"slowlog_length":                       {txt: `Total slowlog`},
 		"start_time_seconds":                   {txt: "Start time of the Redis instance since unix epoch in seconds."},
 		"up":                                   {txt: "Information about the Redis instance"},
+		"connected_clients_details":            {txt: "Total number of connected clients", lbls: []string{"host", "port", "name", "age", "idle", "flags", "qbuf", "qbuffree", "cmd" }},
```

Comment:
metrics descr is wrong, still says `Total number of connected clients`, try something like `Details about connected clients`

Reply:
I don't really want details as much as I want to export number clients sharing labels

Reply:
Sure but other people might find the details useful and/or want to aggregate by host instead of name so this would be useful information.